src/super_gradients/training/transforms/pipeline_adaptors.py

```diff
@@ -39,6 +42,18 @@ def __call__(self, sample, *args, **kwargs):
             self.sample_type = SampleType.DETECTION
         elif isinstance(sample, SegmentationSample):
             self.sample_type = SampleType.SEGMENTATION
+        elif isinstance(sample, PoseEstimationSample):
+            self.sample_type = SampleType.POSE_ESTIMATION
+
+            from albumentations.augmentations import HorizontalFlip
+
+            if any(isinstance(t, HorizontalFlip) for t in self.composed_transforms):
+                raise TypeError(
+                    "`HorizontalFlip` from Albumentation is not supported. "
+                    "Please use the `KeypointsRandomHorizontalFlip` from SuperGradients instead.\n"
+                    "Note: You should set it like other SuperGradients transforms, and not like other Albumentations."
```

Comment:
Can we maybe print an example how it is supposed to look? I think users would be having a hard time figuring it out by themselves. Just a small code or YAML snippet to show how it meant to look would be super helpful.

Reply:
Like this ?

```
TypeError: `HorizontalFlip` from Albumentation is not supported. Please use the `KeypointsRandomHorizontalFlip` from SuperGradients instead.
Note: You should set it like other SuperGradients transforms, and not like other Albumentations.

Example:

FROM 
   > transforms:
   >    - Albumentations:
   >        Compose:
   >            transforms:
   >                - HorizontalFlip:
   >                    p: 1

TO 
   > transforms:
   >    - KeypointsRandomHorizontalFlip:
   >        prob: 1
   >        flip_index: [ 0, 2, 1, 4, 3, 6, 5, 8, 7, 10, 9, 12, 11, 14, 13, 16, 15]
   >        # Note these indexes are COCO-specific. If you're using a different dataset, you will need to change these accordingly.
   >        # The `flip_index` array defines pairs of keypoints to exchange during a horizontal flip. This ensures accurate mapping of corresponding keypoints on mirrored body parts after flipping.

```

Reply:
Yea, that would be great exception tip!